src/components/ResourceLabel.tsx

```diff
@@ -1,16 +1,18 @@
 import { FC } from "react";
 import ResourceIcon, { ResourceIconType } from "./ResourceIcon";
+import classNames from "classnames";
 
 interface Props {
   type: ResourceIconType;
   value: string;
   bold?: boolean;
+  fullWidth?: boolean;
```

Comment:
better call this prop `truncate` instead `fullWidth` I rate, right now we would truncate the content if `fullWidth` is true and that's the opposite behaviour in my mind.

Also, the prop should be set to default `true`. When adjusting existing parameters for functions/components, we should always consider how it could affect the rest of the codebase. The current change will result in pre-existing truncation to be removed everywhere else in the UI.